packages/console/src/components/Executions/TaskExecutionsList/TaskExecutionObservability.tsx

```diff
@@ -0,0 +1,98 @@
+import { LargeLoadingSpinner } from 'components/common/LoadingSpinner';
+import { withRouteParams } from 'components/common/withRouteParams';
+import {
+  NodeExecution,
+  WorkflowExecutionIdentifier,
+} from 'models/Execution/types';
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import { useExternalConfigurationContext } from 'basics/ExternalConfigurationProvider';
+import { useNodeExecution } from 'components/hooks/useNodeExecution';
+import { WaitForData } from 'components/common/WaitForData';
+import { TaskExecutionDetailsAppBarContent } from './TaskExecutionDetailsAppBarContent';
+
+export interface ExecutionDetailsRouteParams {
```

Comment:
I think we're going to need a "view" (enum?) here too; the links need to be able to land on a specific tab within the attempt.